aspnetcore/tutorials/publish-to-azure-webapp-using-vs.md

```diff
@@ -162,39 +149,31 @@ Visual Studio will publish your app to Azure and launch the cloud app in your br
 
 ### Update the app
 
-* Edit the `Views/Home/About.cshtml` Razor view file and change its contents. For example:
-
-<!-- literal_block {"ids": [], "linenos": false, "xml:space": "preserve", "language": "html", "highlight_args": {"hl_lines": [7]}} -->
+* Edit the `Pages/About.cshtml` Razor page and change its contents. For example, you can modify the paragraph to say "Hello ASP.NET Core!":
 
-```html
-@{
-       ViewData["Title"] = "About";
-   }
-   <h2>@ViewData["Title"].</h2>
-   <h3>@ViewData["Message"]</h3>
+    [!code-html[About](publish-to-azure-webapp-using-vs/sample/about.cshtml?highlight=9&range=1-9
 
-   <p>My updated about page.</p>
-   ```
-
-* Right-click on the project and tap **Publish...** again
+* Right-click on the project and select **Publish...** again.
 
 ![Contextual menu open with Publish link highlighted](publish-to-azure-webapp-using-vs/_static/pub.png)
 
-* After the app is published, verify the changes you made are available on Azure
+* After the app is published, verify the changes you made are available on Azure.
+
+![Verify task is complete](publish-to-azure-webapp-using-vs/_static/final.png)
 
 ### Clean up
 
 When you have finished testing the app, go to the [Azure portal](https://portal.azure.com/) and delete the app.
 
-* Select **Resource groups**, then tap the resource group you created
+* Select **Resource groups**, then select the resource group you created.
 
 ![Azure Portal: Resource Groups in sidebar menu](publish-to-azure-webapp-using-vs/_static/portalrg.png)
 
-* In the **Resource group** blade, tap **Delete**
+* In the **Resource group** blade, select **Delete**.
```

Comment:
Resource group --> Resource groups

Reply:
For the sake of clarity, maybe we should tell them to click the ellipsis and select "Delete resource group"?